### DISCUSSION TITLE
Non-x86_64 support

### DISCUSSION THREAD

**Discussion by trentwiles** [plausible/analytics#961] [Self-Hosted Support] [answered]
Does the project have plans for non x86_64 architectures? I'd love to run this on a raspberry pi.

**Comment by metmarkosaric** [comment, marked as answer]
Raspberry Pi is ARM architecture. Clickhouse, our main database dependency, does not support ARM. So unfortunately we cannot support ARM either.

**Reply by KrastanD** [reply]
Seems like they support it now [Link](https://clickhouse.com/docs/en/development/build-cross-arm/) Does this bring ARM support back into the conversation?

**Reply by bhavin192** [reply]
I think we might have it soon, maybe next release? 
Checkout this discussion: https://github.com/plausible/analytics/discussions/903#discussioncomment-1968450
- Container image build for ARM needs to be enabled.
- bytemark/smtp needs to be replaced with an image having ARM build.
